model/daal4py/dffml_model_daal4py/daal4pylr.py

```diff
@@ -114,6 +175,7 @@ async def predict(
             predict = self.pd.DataFrame(feature_data, index=[0])
             preds = self.lm_predictor.compute(predict, self.lm_trained)
             target = self.parent.config.predict.name
-            record.predicted(target, preds.prediction, float("nan"))
+            prediction = preds.prediction
+            record.predicted(target, prediction, float("nan"))
```

Comment:
Sorry, I was not detailed enough. My concern was that the result of prediction should not be an array. But it should be a single value. If daal4py provides and array, and we've asked it to predict a single value, then we need to use element 0 of that array as the value for the prediction

Reply:
Since the user will be expecting a single predicted value, not an array of values. We need to hide the internal weirdness of daal4py

Reply:
Ah I see. Thanks.